rfcs/0110-Best-practices-for-testing-and-credentials.md

```diff
@@ -0,0 +1,99 @@
+# RFC 110 - Best Practices for testing and Credentials
+* Comments: [#110](https://api.github.com/repos/taskcluster/taskcluster-rfcs/issues/110)
+* Proposed by: @djmitche
+
+# Summary
+
+Taskcluster repositories should pass their tests "out of the box": `git clone`, `yarn`, and `yarn test` should result in a successful test run.
```

Comment:
s/`yarn`/`yarn install --frozen-lockfile`
I had problems in the past with docker-worker by forgetting `--frozen-lockfile`